x-pack/plugin/src/test/resources/rest-api-spec/test/ml/inference_crud.yml

```diff
@@ -569,20 +549,6 @@ setup:
   - match: { count: 12 }
   - match: { trained_model_configs.0.model_id: "a-regression-model-1" }
 ---
-"Test get models with include total feature importance":
```

Comment:
See org.elasticsearch.xpack.ml.integration.TrainedModelProviderIT for coverage there